backend/shop/tests/test_views.py

```diff
@@ -504,7 +505,7 @@ def setUp(self):
         perm_change_product = Permission.objects.get(codename="change_product")
         self.user_product_editor.user_permissions.add(perm_change_product)
 
-        self.url_all = reverse("product_get_all")
+        self.url_all = reverse("product_list")
```

Comment:
be consistent, `        self.url_list = reverse("order-list")` one is with `-` and another with `_`.
Recommendation: use `-`